homeassistant/components/samsungtv/media_player.py

```diff
@@ -6,6 +6,7 @@
 from samsungctl import exceptions as samsung_exceptions, Remote as SamsungRemote
 import voluptuous as vol
 import wakeonlan
+from websocket._exceptions import WebSocketException
```

Comment:
Weird that this package name is `_exceptions`. This usually implies that its internal ?

Reply:
Yes strange, but that's the only way to catch it.